data/NonsoBarn.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Nonso Barn",
+  "type": "personal",
+  "bio": "Web Developer. Web3 enthusiast, Games & Tech aficionado"
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/NonsoBarn",
+      "icon": "FaTwitter"
+    }
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/NonsoBarn",
+      "icon": "FaGitHub"
```

The h should be small in Github:
```suggestion
      "icon": "FaGithub"
```